### PR TITLE
Update STIG profile names

### PR DESCRIPTION
**Pull request by shawndwells**
**STIG Profile Cleanup**
- Adds ``stig-rhel7-disa`` profile, aligning only to DISA selected controls
- Update stylesheet headers, removing "PreRelease STIG" language
- Various SRG and RHEL-07-* mappings for rules selected in ``stig-rhel7-disa``
- Updated XCCDF profile description
- Add DISA STIG tables for SRG (https://github.com/OpenSCAP/scap-security-guide/pull/1949/commits/68602e26c0a7887380bc74643064230a7c2e9eb9)
- Add DISA STIG tables for NIST (https://github.com/OpenSCAP/scap-security-guide/pull/1949/commits/492a7d74d8afc7390e988ecd9a110b6e513f4137)
  
**USGCB submission Prep**
  - Renames ``ospp-rhel7-server`` to ``ospp-rhel7``
  - Incorporated ``nist-cl-il-al``, ``stig-rhel7-server-upstream`` profiles into ``ospp-rhel7``

### CODE REVIEW

**Comment by shawndwells** [comment]
ready to review

File: docs/scap-security-guide.8 [hunk 1]
```diff
@@ -231,14 +232,6 @@ This discussion is currently being held on open-scap-list@redhat.com and
...
 
-.I nist-cl-il-al
-.RS
-The CNSSI 1253 Low/Low/Low Control Baseline for Red Hat Enterprise Linux 7 Profile
-follows the Committee on National Security Systems Instruction (CNSSI) No. 1253,
-"Security Categorization and Control Selection for National Security Systems"
-on security controls to meet low confidentiality, low integrity, and low assurance."
-.RE
-
 .I ospp-rhel7-server
```
Comment (yuumasato) [inline docs/scap-security-guide.8]: Should be changed to `ospp-rhel7`

**Comment by shawndwells** [comment]
- Manpage updated (title and description)
- XCCDF profile description updated

**Comment by mpreisler** [comment]
ACK

**Pull request merged**